src/app/payment/paymentMethod/StripeUPEPaymentMethod.tsx

```diff
@@ -29,9 +31,14 @@ const StripeUPEPaymentMethod: FunctionComponent<StripePaymentMethodProps & WithI
     const initializeStripePayment: HostedWidgetPaymentMethodProps['initializePayment'] = useCallback(async (options: PaymentInitializeOptions) => {
         return initializePayment({
             ...options,
-            stripeupe: { containerId },
+            stripeupe: {
+                containerId,
+                onError: (error: any) => {
```

Comment:
i think you can just assign `onUnhandledError` to `onError` as there is no more logic on to it, also onUnhandledError should never be undefined as you are setting a default `noop` function
```suggestion
                onError: onUnhandledError
```